### PR TITLE
bzr-colo 0.4.0 (new formula)

### PR DESCRIPTION
**Pull request by fmccann**
Git style colocated branches for Bazaar.

### CODE REVIEW

File: Library/Formula/bzr-colo.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  bottle :unneeded
+  depends_on "bazaar"
+
+  def install
+    (share/"bazaar/plugins/colo").install Dir["*"]
+  end
+
+  test do
+    edit_file = "echo \"change\" >> foo.txt"
```
Comment (DomT4) [inline Library/Formula/bzr-colo.rb]: I'm not sure it's not better to just inline this.

**Comment by fmccann** [comment]
I did originally have this inline, but that violates the style guidelines in brew audit - is there a way to inline this?

**Comment by DomT4** [comment]
What did `audit` complain about with it inline?

**Comment by fmccann** [comment]
You know what, I tried it again and it didn't complain this time.

**Comment by fmccann** [comment]
Ok, I've put the non-inline style back as it's failing the audit on the server - not sure why it passed on my box sometimes and not others. Same problem in the bzr-rewite branch - neither backticks nor single quotes work either. They either break the test or they break the audit.

I'm open to suggestions :D

File: Library/Formula/bzr-colo.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    system "bzr", "whoami", "Homebrew"
+    system "bzr", "colo-init", "foo"
+
+    cd "foo" do
+      # Checkout a working tree
+      system "bzr", "colo-checkout", "trunk"
+
+      cd "trunk" do
+        # Commit a change in the trunk branch
+        system modify_file
```
Comment (mikemcquaid) [inline Library/Formula/bzr-colo.rb]: Use `(testpath/"foo.txt").write "change"`) instead.

Comment (fmccann) [inline Library/Formula/bzr-colo.rb]: def write(content, *open_args)
    raise "Will not overwrite #{self}" if exist?
    dirname.mkpath
    open("w", *open_args) { |f| f.write(content) }
  end


It looks like I can’t write to the same file twice.

- Fred McCann
fred@sharpnoodles.com

**Comment by apjanke** [comment]
> I'm open to suggestions :D

Homebrew's code extends `Pathname`, so you could do this:

```
file_path = (testpath/"foo/trunk/foo.txt")
...
file_path.write("change")
...
file_path.open("a") { |f| f.puts "change" }
```

Could you also squash this PR to a single commit?

**Comment by fmccann** [comment]
Is the rebased version of this acceptable?

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/a6f127b2e440c86db902290f57b40bcbf8a53890. Thank you once more @fmccann!

**Pull request closed without merging**